types/fs-extra/index.d.ts

```diff
@@ -158,32 +160,32 @@ export function futimes(fd: number, atime: Date, mtime: Date, callback: (err: No
 
-export function lchown(path: string | Buffer, uid: number, gid: number, callback: (err: NodeJS.ErrnoException) => void): void;
-export function lchown(path: string | Buffer, uid: number, gid: number): Promise<void>;
+export function lchown(path: PathLike, uid: number, gid: number, callback: (err: NodeJS.ErrnoException) => void): void;
+export function lchown(path: PathLike, uid: number, gid: number): Promise<void>;
 
-export function link(srcpath: string | Buffer, dstpath: string | Buffer, callback: (err: NodeJS.ErrnoException) => void): void;
-export function link(srcpath: string | Buffer, dstpath: string | Buffer): Promise<void>;
+export function link(existingPath: PathLike, newPath: PathLike, callback: (err: NodeJS.ErrnoException) => void): void;
+export function link(existingPath: PathLike, newPath: PathLike): Promise<void>;
 
-export function lstat(path: string | Buffer, callback: (err: NodeJS.ErrnoException, stats: Stats) => any): void;
-export function lstat(path: string | Buffer): Promise<Stats>;
+export function lstat(path: PathLike, callback: (err: NodeJS.ErrnoException, stats: Stats) => any): void;
+export function lstat(path: PathLike): Promise<Stats>;
 
 /**
  * Asynchronous mkdir - creates the directory specified in {path}.  Parameter {mode} defaults to 0777.
  *
  * @param callback No arguments other than a possible exception are given to the completion callback.
  */
-export function mkdir(path: string | Buffer, callback: (err: NodeJS.ErrnoException) => void): void;
+export function mkdir(path: PathLike, callback: (err: NodeJS.ErrnoException) => void): void;
 /**
  * Asynchronous mkdir - creates the directory specified in {path}.  Parameter {mode} defaults to 0777.
  *
  * @param callback No arguments other than a possible exception are given to the completion callback.
  */
-export function mkdir(path: string | Buffer, mode: number | string, callback: (err: NodeJS.ErrnoException) => void): void;
-export function mkdir(path: string | Buffer): Promise<void>;
+export function mkdir(path: PathLike, options: Mode | fs.MakeDirectoryOptions | null, callback: (err: NodeJS.ErrnoException) => void): void;
+export function mkdir(path: PathLike): Promise<void>;
 
-export function open(path: string | Buffer, flags: string | number, callback: (err: NodeJS.ErrnoException, fd: number) => void): void;
-export function open(path: string | Buffer, flags: string | number, mode: number, callback: (err: NodeJS.ErrnoException, fd: number) => void): void;
-export function open(path: string | Buffer, flags: string | number, mode?: number): Promise<number>;
+export function open(path: PathLike, flags: string | number, callback: (err: NodeJS.ErrnoException, fd: number) => void): void;
+export function open(path: PathLike, flags: string | number, mode: Mode, callback: (err: NodeJS.ErrnoException, fd: number) => void): void;
+export function open(path: PathLike, flags: string | number, mode?: Mode | null): Promise<number>;
```

Comment:
```suggestion
export function open(path: PathLike, flags: string | number, mode?: Mode | null): Promise<number>;
```
can you point usage/source with this param allowing additional type: null?

Reply:
Based on equivalent fs method: https://github.com/NotWoods/DefinitelyTyped/blob/fs-extra-update/types/node/fs.d.ts#L1167

Reply:
thank you!